Assets/MRTK/Core/Utilities/Gltf/Schema/GltfAnimationSampler.cs

```diff
@@ -27,13 +28,33 @@ public class GltfAnimationSampler : GltfProperty
         /// interpolation is `\"STEP\"`, animated value remains constant to the value
         /// of the first point of the timeframe, until the next timeframe.
         /// </summary>
-        public GltfInterpolationType interpolation = GltfInterpolationType.LINEAR;
+        public GltfInterpolationType Interpolation { get; set; }
+
+        [SerializeField]
+        private string interpolation = null;
 
         /// <summary>
         /// The index of an accessor, containing keyframe output values. Output and input
         /// accessors must have the same `count`. When sampler is used with TRS target,
         /// output accessors componentType must be `FLOAT`.
         /// </summary>
         public int output;
```

Comment:
There seem to be many other instances of the proper casing that have not been touched.  Considering the change is to fix serialisation, are these others not affected?